src/tictactoe/game_runner.clj

```diff
@@ -121,24 +104,164 @@
-  (chosen-game-options choices)
-  choices)))
+      (chosen-game-options choices)
+      choices)))
+
+(defn ^:private setup-players 
+  [choices]
+  [{:type :human :marker (choices :player1)} {:type :human :marker (choices :player2)}])
+
+(defn ^:private make-board 
+  [board-choice]
+  (if (= "3x3" board-choice)
+    (vec (take 9 (range)))
+    (vec (take 16 (range)))))
+
+(defn ^:private empty-space? [space]
+  (number? space))
+
+(defn ^:private available-spaces
+  [board]
+  (filter #(empty-space? %) board))
+
+(defn ^:private to-integer 
+  [choice]
+  (try
+    (Integer. (re-find #"^\d+$" choice))
+    (catch Exception e nil)))
+
+(defn ^:private place-move 
+  [spot marker board]
+  (assoc board spot marker))
+
+(defn ^:private get-human-move
+  [available-spaces]
+  (println "\nPlease pick an available cell: ")
+  (let [choice (to-integer (get-input))]
+    (if (and choice (some #(= choice %) available-spaces))
+      choice
+      (recur available-spaces))))
+
+(defn ^:private next-move
+  [current-player board]
+  (if (= (current-player :type) :human)
+    (get-human-move (available-spaces board))))
+
+(defn ^:private row-divider
+  [length]
+  (->> 
+    "--"
+    (repeat (* length length))
+    (apply str)))
+
+(defn ^:private board-dividers
+  [number grid-size]
+  (cond (>= number (dec (* grid-size grid-size))) 
+          nil
+        (= (dec grid-size) (mod number grid-size)) 
+          (str "\n"(row-divider grid-size)"\n")
+        :else "|"))
+
+(defn ^:private grid-size
+  [board]
+  (math/sqrt (count board)))
+
+(defn ^:private clear-screen
+  []
+  (print (str (char 27) "[2J"))
+  (print (str (char 27) "[;H")))
+
+(defn ^:private display-board 
+  [board]
+  (println (apply str (map #(str "  "(board %)"  " (board-dividers % (grid-size board)))
+                            (range (count board))))))
+
+(defn ^:private rows 
+  [board]
+  (partition (grid-size board) (range (count board))))
+
+(defn ^:private columns 
+  [board]
+  (apply map vector (rows board)))
+
+(defn ^:private left-diagonal
+  [board]
+  (reduce #(conj %1 (+ 0 (* %2 (inc (grid-size board))))) 
+          [] (range (grid-size board))))
+
+(defn ^:private right-diagonal
+  [board]
+  (let [right-corner (dec (grid-size board))]
+  (reduce #(conj %1 (+ right-corner (* %2 right-corner)))
+          [right-corner] (range 1 (grid-size board)))))
+
+(defn ^:private diagonals
+  [board]
+    [(left-diagonal board) 
+    (right-diagonal board)])
+
+(defn ^:private winning-combinations 
+  [board]
+  (apply concat [(rows board) (columns board) (diagonals board)]))
+
+(defn ^:private has-three-in-a-row?
+  [board combo]
+  (= 1 (count (set (map #(board %) combo)))))
+
+(defn ^:private winner-exists?
+  [board combo]
+  (if (has-three-in-a-row? board combo)
+    (board (first combo))
+    nil))
+
+(defn ^:private winner? [board]
+  (some #(winner-exists? board %) (winning-combinations board)))
+
+(defn ^:private game-results [board]
+  (let [winner (winner? board)]
```

Comment:
Generally functions ending in `?` suggest a predicate returning `true` or `false`. I would suggest renaming this function to `winner` and `winner?` being implemented as `(some? (winner board))`